pkg/webhook/resources/setting/validator.go

```diff
@@ -1483,3 +1493,15 @@ func validateAdditionalGuestMemoryOverheadRatio(newSetting *v1beta1.Setting) err
 func validateUpdateAdditionalGuestMemoryOverheadRatio(_ *v1beta1.Setting, newSetting *v1beta1.Setting) error {
 	return validateAdditionalGuestMemoryOverheadRatio(newSetting)
 }
+
+func (v *settingValidator) isSingleNode() (bool, error) {
+	nodes, err := v.nodeCache.List(labels.Everything())
+	if err != nil {
+		return false, err
+	}
+	if len(nodes) == 1 {
```

Comment:
```suggestion
	return len(nodes) == 1, nil
```
Is it more readable/clean?

Reply:
yes, thanks for the comment.